org.scala-ide.sdt.aspects/src/scala/tools/eclipse/contribution/weaving/jdt/jcompiler/BuildManagerAspect.aj

```diff
@@ -20,6 +24,14 @@ public aspect BuildManagerAspect {
     this(bm);
 
   /**
+   * Point cut on {@link AbstractImageBuilder#addAllSourceFiles(ArrayList)}
```

Comment:
These Aspects should be seriously documented, for future it is important to know what they are doing and why we need them.